qiskit/transpiler/preset_passmanagers/__init__.py

```diff
@@ -202,6 +204,15 @@ def generate_preset_pass_manager(
     Raises:
         ValueError: if an invalid value for ``optimization_level`` is passed in.
     """
+    if translation_method == "unroller":
+        warnings.warn(
+            "The 'unroller' translation_method plugin has been deprecated and will be removed "
+            "in a future release. Instead you should use the default 'translator' method or "
+            "another plugin",
```

Comment:
same suggestion as above:

```suggestion
            "The 'unroller' translation_method plugin is deprecated as of qiskit-0.45.0 and "
            "will be removed in a future release. Instead you should use the default "
            "'translator' method or another plugin.",
```